reporting/sending_to_dhis2.md

```diff
@@ -0,0 +1,20 @@
+### Sending Reports to DHIS2
+
+1. Set the DHIS2 UUID for the facility that is sending the data by:
+
+   * Got to Legacy System Administration-&gt;Mantainance-&gt;Settings-&gt;Ugandaemr.
+     > ![](../assets/facilityUUID.PNG)
+
+2. Set the DHIS2 Server URL, username and password by:
+
+   * Got to Legacy System Administration-&gt;Mantainance-&gt;Settings-&gt;Ugandaemrsync
+
+   * Save the Above settings
+
+3. Download the Section 4 HTS Report in JSON Format
```

Comment:
@slubwama This is not download, rather Generate the report (can you add a screenshot)